PyDSS/cli/run_server.py

```diff
@@ -0,0 +1,56 @@
+"""
+CLI to run the PyDSS server
+"""
+
+from PyDSS.api.server import pydss_server
```

Comment:
This causes a runtime error. Did you forget to add `server.py` to the repo?
```
ModuleNotFoundError: No module named 'PyDSS.api.server'
```

Reply:
Accidiently pulled from the wrong branch. NAERM code got merged. Fixed this issue yesterday